docs/pages/tutorial/create-your-first-app.mdx

```diff
@@ -32,16 +32,18 @@ It will create a new project directory and install all the necessary dependencie
 <Terminal
   cmd={[
     '# Create a project named StickerSmash',
-    '$ npx create-expo-app StickerSmash',
+    '$ npx create-expo-app --template blank StickerSmash',
     '',
     '# Navigate to the project directory',
     '$ cd StickerSmash',
   ]}
-  cmdCopy="npx create-expo-app StickerSmash && cd StickerSmash"
+  cmdCopy="npx create-expo-app --template blank StickerSmash && cd StickerSmash"
 />
 
 This command will create a new directory for the project with the name: **StickerSmash**.
 
+> With the release of SDK 51, we updated the default template to use [Expo Router](/router/introduction/). This tutorial only has a single screen, so we create the app with the `blank` template instead. After you complete the tutorial, check out the [Expo Router guide](/router/introduction/) to see how to create an app with many screens.
+
```

Comment:
Let's remove this callout. We can instead add steps to explain about Expo Router or new template at the end of the tutorial such as on this page: https://docs.expo.dev/tutorial/follow-up/. (Ideally, link the new get started section on the "Follow up" page).

New/beginners will overlook this at this point of the tutorial and then forget about this information.

Ideally, I'd like us to explain the `--template` part after the running command. This is a good point to learn something new about `create-expo-app` and one of its feature (templates). Let's add something like this:

```md
This command will create a new directory for the project with the name **StickerSmash**. The `create-expo-app` has a `--template` option, which we can use to create a new project with different pre-installed libraries and set it up. In this case, we're using the `blank` with the minimum required libraries installed. We'll continue to add more dependencies throughout this tutorial when we need them.
```